File: FinBoard.Services/DTOs/Snapshot/SnapshotDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FinBoard.Services.DTOs.Move
{
    public class SnapshotDto
    {
        public Guid SnapshotId { get; set; }
        public Guid ResourceId { get; set; }
        public float Amount { get; set; }
        public DateTime DateOfChange { get; set; }
    }
}

File: client/src/app/_models/snapshotModels/snapshotDto.ts
export class SnapshotDto{
    dateOfSnapshot: Date;
    resourceId: string;
    amount: number;

    constructor(date: Date, resourceId:string, amount:number){
        this.dateOfSnapshot = date;
        this.amount = amount;
        this.resourceId = resourceId;
    }

}
